.github/workflows/publish-commit-bottles.yml

```
@@ -0,0 +1,118 @@
+name: Publish and commit bottles
```

```suggestion
name: Publish casks
```

and rename the file too, I reckon.